File: module-rds-repo/src/main/java/entity/Screen.java
package entity;

import java.util.ArrayList;
import java.util.List;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.OneToMany;
import lombok.AllArgsConstructor;
import lombok.NoArgsConstructor;
import lombok.experimental.SuperBuilder;

@Entity
@SuperBuilder
@NoArgsConstructor
@AllArgsConstructor
public class Screen extends BaseEntity{

	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	@Column(name = "screen_id")
	private Long screen_id;

	@OneToMany(mappedBy = "screen")
	private List<Seats> seats = new ArrayList<>();

	private String name;
}

File: domain/src/main/kotlin/io/github/jehwanyoo/redis_1st/entity/Screen.kt
package io.github.jehwanyoo.redis_1st.entity

data class Screen(
    val id: Long,       // 고유 ID
    val name: String,   // 상영관 이름
    val row: Int,       // 좌석 행 수
    val column: Int,    // 좌석 열 수
)
